src/client.ts

```diff
@@ -14,144 +14,217 @@ import {
+   */
+  get axiosInstance(): AxiosInstance {
+    return this.instanceV1
   }
 
+  /**
+   * Retrieves the price for a given request.
+   * @param {InscriptionPriceRequest} priceRequest - The request object for price calculation.
+   * @returns {Promise<InscriptionPriceResponse>} A promise resolving with the price response.
+   */
   async getPrice(
     priceRequest: InscriptionPriceRequest
   ): Promise<InscriptionPriceResponse> {
     return this.instanceV1.get(`/price`, {
       params: priceRequest,
-    });
+    })
   }
 
+  /**
+   * Creates an inscription order.
+   * @param {InscriptionOrderRequest} order - The request object for creating the order.
+   * @returns {Promise<InscriptionOrder>} A promise resolving with the created order.
+   */
   async createOrder(order: InscriptionOrderRequest): Promise<InscriptionOrder> {
-    return this.instanceV1.post(`/order`, order);
+    return this.instanceV1.post(`/order`, order)
   }
 
+  /**
+   * Retrieves an inscription order by ID.
+   * @param {string} id - The ID of the order to retrieve.
+   * @returns {Promise<InscriptionOrder>} A promise resolving with the retrieved order.
+   */
   async getOrder(id: string): Promise<InscriptionOrder> {
     return this.instanceV1.get(`/order`, {
       params: { id },
-    });
+    })
   }
 
+  /**
+   * Creates a collection for inscriptions.
+   * @param {InscriptionCollectionCreateRequest} collection - The request object for creating the collection.
+   * @returns {Promise<InscriptionCollectionCreateResponse>} A promise resolving with the created collection response.
+   */
   async createCollection(
     collection: InscriptionCollectionCreateRequest
   ): Promise<InscriptionCollectionCreateResponse> {
     // modify normal json to valid form data for files
-    let plainObject = Object.assign({ ...collection });
-    let files = collection?.files;
+    let plainObject = Object.assign({ ...collection })
+    let files = collection?.files
     for (let index in files) {
-      let file: any = files[index];
-      let keys = Object.keys(file);
+      let file: any = files[index]
+      let keys = Object.keys(file)
       for (let key in keys) {
-        let propName = keys[key];
-        plainObject[`files[${index}][${propName}]`] = file[propName];
+        let propName = keys[key]
+        plainObject[`files[${index}][${propName}]`] = file[propName]
       }
     }
-    delete plainObject.files;
-    let data = qs.stringify(plainObject);
+    delete plainObject.files
+    let data = qs.stringify(plainObject)
     // modify normal json to valid form data for files
 
     let config = {
-      method: "post",
+      method: 'post',
       maxBodyLength: Infinity,
-      url: this.instanceV1.getUri() + "/collectioncreate",
+      url: this.instanceV1.getUri() + '/collectioncreate',
       headers: {
-        "Content-Type": "application/x-www-form-urlencoded",
+        'Content-Type': 'application/x-www-form-urlencoded',
```

Comment:
this doesn't look right. we accept json in the backend.

Reply:
We already had  "Content-Type": "application/x-www-form-urlencoded" there is no change here.